docs/references/cluster-configuration.md

```diff
@@ -40,9 +39,11 @@ tenants:
     - ci
 ```
 
+## Optional parameters
+
 ### `node_count`
 
-The number of machines this Opstrace cluster is supposed to be comprised of.
+The number of underlying instances to use.
```

Comment:
Let's revert that part of the change that removes the notion of "machines". 

"instance" is a highly generic term, and not everyone immediately associates it with the number of VMs/nodes/machines.

Sorted by clarity:
1. machine count / node count
2. instance count

Reply:
As long as we pick from (1) we're good, I thought I had said that :D